routes/rendering-info.js

```diff
@@ -182,27 +197,28 @@ const postRenderingInfoRoute = {
     tags: ['api', 'editor']
   },
   handler: function(request, reply) {
-    let toolRuntimeConfig = {};
-    if (request.payload.hasOwnProperty('toolRuntimeConfig')) {
-      toolRuntimeConfig = request.payload.toolRuntimeConfig;
+    let requestToolRuntimeConfig = {};
```

Comment:
Would name it requestToolRuntimeConfig in the other handler method, too. So it's clear that you handover this special kind of TRC to getRenderingInforForID and only there it will be merged to compiled TRC.